docs/themes/book/i18n/en.yaml

```diff
@@ -12,3 +12,5 @@
 
 - id: bookSearchConfig
   translation: '{ cache: true, split: /[^a-zа-яё0-9\w]/gi }'
```

Comment:
This is our workaround for pages with mixed language.

Reply:
I've found a solution without this hack, but it requires to change pages structure.
I'll make a proposal. Separate languages have some drawbacks.

Reply:
Do we need to keep .yaml files for all languages we do not support?

Reply:
My motivation was to reduce the diff in theme updates. They are cheap.